geoportailv3/static/js/draw/drawdirective.js

```diff
@@ -318,15 +339,17 @@ app.DrawController.prototype.onChangeActive_ = function(event) {
 app.DrawController.prototype.onDrawEnd_ = function(event) {
   var feature = event.feature;
   feature.set('name', 'element ' + (++this.featureSeq_));
+  feature.set('description', '');
   feature.set('__editable__', true);
   feature.set('color', '#ed1c24');
   feature.set('opacity', 0.2);
   feature.set('stroke', 1.25);
   feature.set('size', 10);
   feature.set('angle', 0);
-  feature.set('is_label', false);
   feature.set('linestyle', 'plain');
   feature.set('symbol_id', 'circle');
+  this.drawLabel.getActive() ?
+      feature.set('isLabel', true) : feature.set('isLabel', false);
```

Comment:
By the way we used the underscore notation for feature property names here because those were coming from the server.